src/modules/playRoom/components/ChatBox.js

```diff
@@ -11,25 +29,32 @@ const ChatBox = () => {
         </div>
         <div className="scroll">
           <div className="chats">
-            <div className="chats-client">
-              <strong>nddmanh</strong> vừa đặt cược <strong>100$</strong> vào
-              <strong>Cua</strong>
-            </div>
-            <div className="chats-self">
-              <strong>Bạn</strong> vừa đặt cược <strong>200$</strong> vào
-              <strong>Bầu</strong>
-            </div>
+            {
+              messageList.map((message, index) => {
+                return <div className={message._id === props.userId ? 'chats-self' : 'chats-client'} key={index}>
```

Comment:
Don't use index as key in list, you can generate an uuid for each message

Reply:
I think it still doing fine. But Im gonna fix it later when I have free time.

Reply:
ok